Requirements/Requirements_And_Goals.md

```diff
@@ -0,0 +1,73 @@
+# Requirements and Goals
+
+## Mission
+
+> The Hey, Blue! initiative facilitates moments of meaningful connection between police officers and members of their community by offering intentional opportunities for individuals to meet and share positive experience.
+
+## Goal
+
+To provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. This platform can be hosted on both web and mobile based. 
+
+## Actors
+- Citiens
+- Police officers
+- Charities
+- Retail Organisations
+
+## Functional Requirements
+
+### Hard Requirements 
+ 1. End-Use Ease of use (HR1)
+ 2. Location Tracking (HR2)
+ 3. Connecting with other users (HR3)
+ 4. Uploading to social media (HR4)
+ 5. Tracking engagements
+ 6. Ability to opt in to and out of push Notifications
+ 7. Users can only have one connection per officer per 24hour period
+ 8. Community members can find officers and not the other way around
+ 9. Officers locations shuts off after 15 minutes
+ 10. community members to be able to see how many connections each participating officer has made
+ 11. Allow Hey, Blue! to track all participation by zi cde. 
+
+### Civilian Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create User Profile. (Allow deletion, updating, Hide?)|User registration|
+|2|Update, delete, Hide User Profile.|Profile Service|
+|3| Notify for an interaction with policice office within 10ft distance|Notification and Location|
+|4|Establish connection with the police officer and have a virtual handshake|Handshake|
+|5|Post connections on HeyBlue! social media|right name?|
+|6|Update civilians account/profile with points|PointsCredition|
+|7|*allow civilians to upload the awards program of other business into Hey Blue!*|?|
+|8|Redeem points at participating business for goods|Redeem|
+
+### Police Officer Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create User Profile. (Only work email and load diff prefixs|User registration|
+|2|Update, delete, Hide User Profile.|Profile Service|
+|3| Notify for an interaction request within 10ft|Notification and Location|
+|4|Establish connection with the civilian|Handshake|
+|5|Post connections on HeyBlue! social media(Post immediately or in 5/10mins)|right name?|
+|6|Update officer's account/profile with points|PointsCredition|
+|7|Transfer points to non profit (there is more here)|?|
+|8|ability to upload upcoming events to find officers willing to connect|?|
+|9|Geo location must shut off every 15mins|Location|
+
+### Charity Process
+
+|Id|User Story|Service|
+|--- |--- |---|
+|1|Create Profile.|User registration|
+|2|Update, delete, Hide User Profile.|Profile Service|
+|3|Allow Police officers to connect|Handshake|
```

Comment:
For charity?

Reply:
what's this? didn't understand

Reply:
Allow Police officers to connect is not a requiremnt of a charity is it?

Reply:
It is. according to the charity process in requirement doc

Reply:
seems strange but ok